### ISSUE TITLE
Get class name labels on ssd_mobilenetv2_oidv4

### ISSUE THREAD

**Issue by yieniggu** [tensorflow/models#7955]
How are the classes name sorted in the oidv4 model?

I've downloaded the class names csv file and converted it into a plain txt containing only the names.

Whe I make a test on [this](https://ibb.co/d2y9ycB) image it gives me this predictions, with the labels shown on the image.

Is it problem with sorting the names or with the predictions itself?

Here is my txt file with the 600 classes:

Tortoise
Container
Magpie
Sea turtle
Football
Ambulance
Ladder
Toothbrush
Syringe
Sink
Toy
Organ (Musical Instrument)
Cassette deck
Apple
Human eye
Cosmetics
Paddle
Snowman
Beer
Chopsticks
Human beard
Bird
Parking meter
Traffic light
Croissant
Cucumber
Radish
Towel
Doll
Skull
Washing machine
Glove
Tick
Belt
Sunglasses
Banjo
Cart
Ball
Backpack
Bicycle
Home appliance
Centipede
Boat
Surfboard
Boot
Headphones
Hot dog
Shorts
Fast food
Bus
Boy
Screwdriver
Bicycle wheel
Barge
Laptop
Miniskirt
Drill (Tool)
Dress
Bear
Waffle
Pancake
Brown bear
Woodpecker
Blue jay
Pretzel
Bagel
Tower
Teapot
Person
Bow and arrow
Swimwear
Beehive
Brassiere
Bee
Bat (Animal)
Starfish
Popcorn
Burrito
Chainsaw
Balloon
Wrench
Tent
Vehicle registration plate
Lantern
Toaster
Flashlight
Billboard
Tiara
Limousine
Necklace
Carnivore
Scissors
Stairs
Computer keyboard
Printer
Traffic sign
Chair
Shirt
Poster
Cheese
Sock
Fire hydrant
Land vehicle
Earrings
Tie
Watercraft
Cabinetry
Suitcase
Muffin
Bidet
Snack
Snowmobile
Clock
Medical equipment
Cattle
Cello
Jet ski
Camel
Coat
Suit
Desk
Cat
Bronze sculpture
Juice
Gondola
Beetle
Cannon
Computer mouse
Cookie
Office building
Fountain
Coin
Calculator
Cocktail
Computer monitor
Box
Stapler
Christmas tree
Cowboy hat
Hiking equipment
Studio couch
Drum
Dessert
Wine rack
Drink
Zucchini
Ladle
Human mouth
Dairy Product
Dice
Oven
Dinosaur
Ratchet (Device)
Couch
Cricket ball
Winter melon
Spatula
Whiteboard
Pencil sharpener
Door
Hat
Shower
Eraser
Fedora
Guacamole
Dagger
Scarf
Dolphin
Sombrero
Tin can
Mug
Tap
Harbor seal
Stretcher
Can opener
Goggles
Human body
Roller skates
Coffee cup
Cutting board
Blender
Plumbing fixture
Stop sign
Office supplies
Volleyball (Ball)
Vase
Slow cooker
Wardrobe
Coffee
Whisk
Paper towel
Personal care
Food
Sun hat
Tree house
Flying disc
Skirt
Gas stove
Salt and pepper shakers
Mechanical fan
Face powder
Fax
Fruit
French fries
Nightstand
Barrel
Kite
Tart
Treadmill
Fox
Flag
French horn
Window blind
Human foot
Golf cart
Jacket
Egg (Food)
Street light
Guitar
Pillow
Human leg
Isopod
Grape
Human ear
Power plugs and sockets
Panda
Giraffe
Woman
Door handle
Rhinoceros
Bathtub
Goldfish
Houseplant
Goat
Baseball bat
Baseball glove
Mixing bowl
Marine invertebrates
Kitchen utensil
Light switch
House
Horse
Stationary bicycle
Hammer
Ceiling fan
Sofa bed
Adhesive tape
Harp
Sandal
Bicycle helmet
Saucer
Harpsichord
Human hair
Heater
Harmonica
Hamster
Curtain
Bed
Kettle
Fireplace
Scale
Drinking straw
Insect
Hair dryer
Kitchenware
Indoor rower
Invertebrate
Food processor
Bookcase
Refrigerator
Wood-burning stove
Punching bag
Common fig
Cocktail shaker
Jaguar (Animal)
Golf ball
Fashion accessory
Alarm clock
Filing cabinet
Artichoke
Table
Tableware
Kangaroo
Koala
Knife
Bottle
Bottle opener
Lynx
Lavender (Plant)
Lighthouse
Dumbbell
Human head
Bowl
Humidifier
Porch
Lizard
Billiard table
Mammal
Mouse
Motorcycle
Musical instrument
Swim cap
Frying pan
Snowplow
Bathroom cabinet
Missile
Bust
Man
Waffle iron
Milk
Ring binder
Plate
Mobile phone
Baked goods
Mushroom
Crutch
Pitcher (Container)
Mirror
Personal flotation device
Table tennis racket
Pencil case
Musical keyboard
Scoreboard
Briefcase
Kitchen knife
Nail (Construction)
Tennis ball
Plastic bag
Oboe
Chest of drawers
Ostrich
Piano
Girl
Plant
Potato
Hair spray
Sports equipment
Pasta
Penguin
Pumpkin
Pear
Infant bed
Polar bear
Mixer
Cupboard
Jacuzzi
Pizza
Digital clock
Pig
Reptile
Rifle
Lipstick
Skateboard
Raven
High heels
Red panda
Rose
Rabbit
Sculpture
Saxophone
Shotgun
Seafood
Submarine sandwich
Snowboard
Sword
Picture frame
Sushi
Loveseat
Ski
Squirrel
Tripod
Stethoscope
Submarine
Scorpion
Segway
Training bench
Snake
Coffee table
Skyscraper
Sheep
Television
Trombone
Tea
Tank
Taco
Telephone
Torch
Tiger
Strawberry
Trumpet
Tree
Tomato
Train
Tool
Picnic basket
Cooking spray
Trousers
Bowling equipment
Football helmet
Truck
Measuring cup
Coffeemaker
Violin
Vehicle
Handbag
Paper cutter
Wine
Weapon
Wheel
Worm
Wok
Whale
Zebra
Auto part
Jug
Pizza cutter
Cream
Monkey
Lion
Bread
Platter
Chicken
Eagle
Helicopter
Owl
Duck
Turtle
Hippopotamus
Crocodile
Toilet
Toilet paper
Squid
Clothing
Footwear
Lemon
Spider
Deer
Frog
Banana
Rocket
Wine glass
Countertop
Tablet computer
Waste container
Swimming pool
Dog
Book
Elephant
Shark
Candle
Leopard
Axe
Hand dryer
Soap dispenser
Porcupine
Flower
Canary
Cheetah
Palm tree
Hamburger
Maple
Building
Fish
Lobster
Garden Asparagus
Furniture
Hedgehog
Airplane
Spoon
Otter
Bull
Oyster
Horizontal bar
Convenience store
Bomb
Bench
Ice cream
Caterpillar
Butterfly
Parachute
Orange
Antelope
Beaker
Moths and butterflies
Window
Closet
Castle
Jellyfish
Goose
Mule
Swan
Peach
Coconut
Seat belt
Raccoon
Chisel
Fork
Lamp
Camera
Squash (Plant)
Racket
Human face
Human arm
Vegetable
Diaper
Unicycle
Falcon
Chime
Snail
Shellfish
Cabbage
Carrot
Mango
Jeans
Flowerpot
Pineapple
Drawer
Stool
Envelope
Cake
Dragonfly
Common sunflower
Microwave oven
Honeycomb
Marine mammal
Sea lion
Ladybug
Shelf
Watch
Candy
Salad
Parrot
Handgun
Sparrow
Van
Grinder
Spice rack
Light bulb
Corded phone
Sports uniform
Tennis racket
Wall clock
Serving tray
Kitchen & dining room table
Dog bed
Cake stand
Cat furniture
Bathroom accessory
Facial tissue holder
Pressure cooker
Kitchen appliance
Tire
Ruler
Luggage and bags
Microphone
Broccoli
Umbrella
Pastry
Grapefruit
Band-aid
Animal
Bell pepper
Turkey
Lily
Pomegranate
Doughnut
Glasses
Human nose
Pen
Ant
Car
Aircraft
Human hand
Skunk
Teddy bear
Watermelon
Cantaloupe
Dishwasher
Flute
Balance beam
Sandwich
Shrimp
Sewing machine
Binoculars
Rays and skates
Ipod
Accordion
Willow
Crab
Crown
Seahorse
Perfume
Alpaca
Taxi
Canoe
Remote control
Wheelchair
Rugby ball
Armadillo
Maracas
Helmet

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
